operators/nftopology/PROJECT

```diff
@@ -0,0 +1,16 @@
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
+domain: nephio.org
+layout:
+- go.kubebuilder.io/v3
+projectName: nftopology
+repo: nephio-project/nftopology
```

Comment:
Maybe we should keep it int hat repo? Just for expediency, if there are build issues from moving it?